### PR TITLE
Otel testing

### PR DESCRIPTION
**Pull request by Kuckkuck**
- ceph parsing
- new image registry
 A lot of commits for testing

### CODE REVIEW

File: opentelemetry/chart/templates/smon-filelog.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ metadata:
   annotations:
   name: opentelemetry-collector-logs
   labels:
-    prometheus: "{{ .Values.openTelemetry.prometheus }}"
+    prometheus: infra-frontend
```
Comment (viennaa) [inline opentelemetry/chart/templates/smon-filelog.yaml:12]: I would rather have this specced in the values, this is very static and will involve touching soon.

File: opentelemetry/chart/values.yaml [hunk 2]
```diff
@@ -1,17 +1,33 @@
...
+    create: true
+    certManager:
+      enabled: false
+    autoGenerateCert:
+      enabled: true
+      recreate: false
   manager:
     collectorImage:
-      repository: ghcr.io/cloudoperators/opentelemetry-collector-contrib
+      repository: keppel.eu-de-1.cloud.sap/ccloud/opentelemetry/collector-image
```
Comment (viennaa) [inline opentelemetry/chart/values.yaml:26]: make this configurable. it should default to cloudoperators, we add this as a value in secrets for ccloud deployment.

File: opentelemetry/chart/values.yaml [hunk 3]
```diff
@@ -1,17 +1,33 @@
...
       #repository: ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib
-      tag: main
       #tag: 0.104.0
+      #tag: main
+      tag: latest
```
Comment (viennaa) [inline opentelemetry/chart/values.yaml:30]: can we go with versions from Chart.yaml here? same applies to opentelemetry-operator

**Review by viennaa — CHANGES_REQUESTED** [review]
Thanks for adding this PR. Some comments are inline.

Please squash the commits to at most 5 and give it a meaningful message like

* dashboard added
* crds added
* image location changed

File: opentelemetry/chart/values.yaml (hunk 2 above)
Comment (Kuckkuck) [reply opentelemetry/chart/values.yaml]: switched to docker image as default

**Review by viennaa — APPROVED** [review]
Review took place via phone. Good to go.

**Pull request merged**